packages/terra-responsive-element/src/ResponsiveElement.jsx

```diff
@@ -14,11 +14,11 @@ const propTypes = {
    */
   children: PropTypes.node,
   /**
-   * A callback function invoked when the breakpoint size changes. onChange(<string>breakpoint)
+   * A callback function invoked when the breakpoint size changes.
```

Comment:
Why did we update the proptype description? What you removed describes what the callback should be expecting when the Responsive Element uses it

Reply:
Updated here https://github.com/cerner/terra-core/pull/2864/commits/91a0e30e1e2746bc2e0363a1af4afec879906d7f.